knowledge-base/kb-chartview-howto-export-without-panzoom-scrollbars.md

```diff
@@ -0,0 +1,57 @@
+---
+title: How to export the chart without the pan and zoom scrollbars
```

Comment:
consider using the control's name in the metadata - RadCartesianChart or RadChartView